api/src/main/java/com/graphhopper/util/CustomModel.java

```diff
@@ -58,12 +56,12 @@ public CustomModel(CustomModel toCopy) {
      * especially important for fast landmark queries (hybrid mode). Now this method ensures that all server-side custom
      * models are cached in a special internal cache which does not remove seldom accessed entries.
      */
-    CustomModel useInternalCache() {
+    public CustomModel useInternalCache() {
         this.cached = true;
         return this;
     }
 
-    boolean shouldUseInternalCache() {
+    public boolean shouldUseInternalCache() {
```

Comment:
hmmh, how should we solve this when CustomModel and CustomProfile are in different modules?

Reply:
You mean how can we make it not public?

Reply:
Yes

Reply:
I don't really think it needs to be private. We could prefix it with _ to make it more clear that people who use it should know what they are doing. It was prefixed like this before :laughing:

Reply:
I think we can keep it public and only throw an exception if queryCustomModel has it set to true.